### PR TITLE
[MIRROR]  Atmos cleanup + subtypes

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/36690
--------------------
I cleaned up a bunch of atmos related dvs and created subtypes that should make mapping atmospherics easier and faster.

- Igniters now default to on = FALSE - this makes sense since they start switched off in absolutely all maps except for CentCom and two away missions. I removed the luminosity overrides as well, since igniter luminosity is defined by the power state.
- added mixer/airmix subtype that uses N2STANDARD/O2STANDARD and MAX_OUTPUT_PRESSURE instead of "0.8/0.2" and "4500"
- added freezer/on and heater/on, with the freezer defaulting to the lowest possible temp
- added waste loop specific filter/atmos subtypes (filter/atmos/o2 etc.)

### CODE REVIEW

**Comment by JohnGinnane** [comment]
@Pyko1

**Comment by Pyko1** [comment]
should be done

File: _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm [hunk 1]
```diff
@@ -4839,7 +4835,7 @@
 /obj/machinery/atmospherics/components/binary/pump/on{
 	dir = 8;
 	name = "O2 to Incinerator";
-	target_pressure = 4500
+	target_pressure = MAX_OUTPUT_PRESSURE
```
Comment (praisenarsie) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: change back to 4500

File: _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm [hunk 2]
```diff
@@ -5855,7 +5851,7 @@
 	pixel_y = 24
 	},
 /obj/machinery/atmospherics/components/binary/pump/on{
-	target_pressure = 4500
+	target_pressure = MAX_OUTPUT_PRESSURE
```
Comment (praisenarsie) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: change back to 4500

File: _maps/templates/pirate_ship.dmm [hunk 3]
```diff
@@ -1055,22 +1055,15 @@
 /turf/open/floor/plasteel,
 /area/shuttle/pirate)
 "cv" = (
-/obj/machinery/atmospherics/components/binary/pump{
+/obj/machinery/atmospherics/components/binary/pump/on{
 	dir = 4;
 	name = "N2 Out";
-	on = 1;
-	target_pressure = 4500
+	target_pressure = MAX_OUTPUT_PRESSURE
```
Comment (praisenarsie) [inline _maps/templates/pirate_ship.dmm]: change back to 4500

File: _maps/templates/pirate_ship.dmm [hunk 4]
```diff
@@ -1247,7 +1239,7 @@
 /obj/machinery/atmospherics/components/binary/pump{
 	dir = 8;
 	name = "Port Out";
-	target_pressure = 4500
+	target_pressure = MAX_OUTPUT_PRESSURE
```
Comment (praisenarsie) [inline _maps/templates/pirate_ship.dmm]: change back to 4500

File: _maps/templates/pirate_ship.dmm [hunk 5]
```diff
@@ -1370,7 +1362,7 @@
 /obj/machinery/atmospherics/components/binary/pump{
 	dir = 2;
 	name = "O2 to Incinerator";
-	target_pressure = 4500
+	target_pressure = MAX_OUTPUT_PRESSURE
```
Comment (praisenarsie) [inline _maps/templates/pirate_ship.dmm]: change back to 4500

File: _maps/templates/pirate_ship.dmm [hunk 6]
```diff
@@ -1521,7 +1513,7 @@
 /obj/machinery/atmospherics/components/binary/pump{
 	dir = 8;
 	name = "Plasma to Incinerator";
-	target_pressure = 4500
+	target_pressure = MAX_OUTPUT_PRESSURE
```
Comment (praisenarsie) [inline _maps/templates/pirate_ship.dmm]: change back to 4500

File: _maps/templates/pirate_ship.dmm [hunk 7]
```diff
@@ -1596,10 +1588,9 @@
 	pixel_y = -24
 	},
 /obj/effect/turf_decal/stripes/line,
-/obj/machinery/atmospherics/components/binary/pump{
+/obj/machinery/atmospherics/components/binary/pump/on{
 	dir = 2;
-	on = 1;
-	target_pressure = 4500
+	target_pressure = MAX_OUTPUT_PRESSURE
```
Comment (praisenarsie) [inline _maps/templates/pirate_ship.dmm]: .

File: _maps/templates/pirate_ship.dmm [hunk 8]
```diff
@@ -1670,7 +1661,7 @@
 /obj/machinery/atmospherics/components/binary/pump{
 	dir = 8;
 	name = "Port to Incinerator";
-	target_pressure = 4500
+	target_pressure = MAX_OUTPUT_PRESSURE
```
Comment (praisenarsie) [inline _maps/templates/pirate_ship.dmm]: .

**Comment by Jujumatic** [comment]
narsie are you ok the bot is freaking out

**Comment by Pyko1** [comment]
included the fixes from https://github.com/HippieStation/HippieStation/pull/7401 to this PR

**Review by praisenarsie — APPROVED** [review]

**Pull request merged**